src/lib.rs

```diff
@@ -679,6 +680,13 @@ pub struct TestNetworkConfiguration {
     stored_configuration: StoredConfiguration,
 }
 
+// A new configuration proposal state
+#[derive(Debug)]
+enum ConfigurationProposalState {
+    Uncommited(TestNetworkConfiguration),
```

Comment:
It's `Committed` and `Uncommitted` (double `t` in both).

Reply:
Fixed